tests/actions/buildiso/buildiso_test.py

```diff
@@ -0,0 +1,164 @@
+import os
+
+import pytest
+
+from cobbler import enums
+from cobbler.actions import buildiso
+from cobbler.actions.buildiso.netboot import NetbootBuildiso
+from cobbler.actions.buildiso.standalone import StandaloneBuildiso
+from tests.conftest import does_not_raise
+
+
+@pytest.mark.parametrize(
+    "input_arch,result_binary_name,expected_exception",
+    [
+        (enums.Archs.X86_64, "grubx64.efi", does_not_raise()),
+        (enums.Archs.PPC, "grub.ppc64le", does_not_raise()),
+        (enums.Archs.PPC64, "grub.ppc64le", does_not_raise()),
+        (enums.Archs.PPC64EL, "grub.ppc64le", does_not_raise()),
```

Comment:
Shouldn't this be `grub.ppc64el`?

Reply:
No because (as per the comment in the source file), grub2 can boot all these PowerPC architectures with the same loader.

https://github.com/cobbler/cobbler/blob/3f94a33908e5700e32533635fc6192f9d247d02c/cobbler/actions/buildiso/__init__.py#L205-L207